olad/OladHTTPServer.cpp

```diff
@@ -442,6 +443,15 @@ int OladHTTPServer::JsonServerStats(const HTTPRequest*,
 #endif  // _WIN32
 
   JsonObject json;
+#ifdef OLA_SCREENSHOT_MODE
+  json.Add("hostname", "olad-server");
+  json.Add("instance_name", OLA_DEFAULT_INSTANCE_NAME);
+  json.Add("config_dir", "/etc/ola/");
+  json.Add("ip", "10.79.76.65");
```

Comment:
We should use '***' or something similar rather than making up values.

Reply:
It spells 10.O.L.A in ASCII! I thought semi-genuine data would be more helpful in a screenshot, rather than lots of empty values. It would also mean you could look at the layout and styling realistically.

Reply:
Done